### DISCUSSION TITLE
Any lake formation permissions issues

### DISCUSSION THREAD

**Discussion by gregcsh** [aws/aws-sdk-pandas#652] [Q&A]
Typically with Spark doing a partition overwrite against a table where the location has been registered and perms applied even with super access is denied. This is even when using purge_table as well as other Spark or Pandas comands thru Sagemaker and Glue ETL. If the location and perms are removed from LF it all works great.

PyAthena works great but I have seen some perf issues. **Will data-wrangler work with Lake Formation objects?**

**Comment by jaidisido** [comment]
Lake Formation permissions apply to the IAM entity making the calls. So as long as the IAM role/user used to run Wrangler has enough permissions, I do not expect any issues. Have you experienced specific problems with this?

**Reply by gregcsh** [reply]
So far in all my testing I have not. I just wanted to know if there were any Lake Formation gotchas.

**Reply by gregcsh** [reply]
I have had some perms issues, I opened an Issue on the topic. #663
